pages/api/auth/signup.ts

```diff
@@ -0,0 +1,22 @@
+import { NextApiRequest, NextApiResponse } from 'next'
```

Comment:
This file allows us to add a user to our database, since next-auth only handles signing in existing users. For more details about how Next API routes work, talk to me or check out [this](https://nextjs.org/docs/api-routes/introduction) page